src/services/__mocks__/pncClient.ts

```diff
@@ -0,0 +1,30 @@
+import axios, { AxiosInstance } from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import buildsRequestMock from './builds-mock.json';
+import projectsRequestMock from './projects-mock.json';
+import projectRequestMock from './project-mock.json';
+
+let mock = new MockAdapter(axios);
+
+class ClientMock {
```

Comment:
use `PncClientMock` and change https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/services/pncClient.ts#L7 also to `PncClient`